UI/window-basic-status-bar.cpp

```diff
@@ -6,92 +6,61 @@
 #include "window-basic-main.hpp"
 #include "window-basic-status-bar.hpp"
 #include "window-basic-main-outputs.hpp"
+#include "qt-wrappers.hpp"
+#include "platform.hpp"
+
+#include "ui_StatusBarWidget.h"
+
+#define CONGESTION_UPDATE_SECONDS 10
+#define EXCELLENT_THRESHOLD 0.0f
+#define GOOD_THRESHOLD 0.3333f
+#define MEDIOCRE_THRESHOLD 0.6667f
+#define BAD_THRESHOLD 1.0f
```

Comment:
We should prefer `constexpr` over macros for new code.

Reply:
Updated to use constexpr